### ISSUE TITLE
Not showing the results from SonarCloud.

### ISSUE THREAD

**Issue by CKannas** [mibexsoftware/sonar-bitbucket-plugin#63]
I have a public repository hosted on BitBucket, https://bitbucket.org/CKannas/moreactmoldesign/overview.

I added Sonar BitBucket plugin and the required sonar.json file.
`{
    "sonarHost": "https://sonarcloud.io",
    "sonarProjectKey": "ckannas:moreactmoldesign:default"
}`

I've run SonarQube scanner on my project which uploaded the results on SonarCloud https://sonarcloud.io/dashboard?id=ckannas%3Amoreactmoldesign%3Adefault. 

Though I can not see the results from SonarCloud at the overview page of my project.

I think the webhook for the plugin works ok, it sends a request and the response has error code 204 with empty body.

What's the correct configuration for connecting to SonarCloud?

**Comment by mrueegg** [comment]
Please excuse my late response. The configuration actually looks right. Do you get an error or just don't see any statistics in the Bitbucket cloud repository overview page?

**Comment by CKannas** [comment]
Sorry for my very very late reply.

Just to be sure I re-created my SonarCloud [project](https://sonarcloud.io/dashboard?id=ckannas%3Amoreactmoldesign%3Adefault), but the issue persists.
I can't see the statistics in the [project's overview page](https://bitbucket.org/CKannas/moreactmoldesign/overview).
Similar to previously, the webhook it sends a request and the response has a green (Ok?) error code 204 with an empty body. [Screenshot](http://prntscr.com/hl9kx1)

**Comment by starchx** [comment]
I don't think this will work with sonarcloud.io, as it requires the plugin to be installed on a sonar host. Can author clarify please? Thanks
